### ISSUE TITLE
Support resize handles

### ISSUE THREAD

**Issue by mattlewis92** [mattlewis92/angular2-resizable#10]
Allow specifying an element to act as the drag handle on the element like:
```
<div class="rectangle" [ngStyle]="style" mwl-resizeable (onResizeEnd)="onResizeEnd($event)">
  <img src="http://i.imgur.com/eqzz2dl.gif" style="position: absolute; bottom: 5px; right: 5px;" resize-handle edges="{bottom: true, right: true}"}>
</div>
```
![screen shot 2016-06-14 at 10 26 22](https://cloud.githubusercontent.com/assets/6425649/16037895/7697b418-321a-11e6-95f8-1d8729c61813.png)

**Comment by ivanmayes** [comment]
Love it.  I'm implementing something similar to this functionality: http://styleguide.devbproto.com/styleguide/#3 and would love to see a breakout that would let you attach custom handle elements for dragging.  Will pull this from npm and see if I can figure that out without breaking anything!

**Comment by mattlewis92** [comment]
I've just published 0.1.0 which has this feature, LMK if it works ok for you! 😄
